packages/solid/src/generators/application/lib/project/vite.ts

```diff
@@ -0,0 +1,60 @@
+import { NormalizedSchema } from '../../schema';
+import { joinPathFragments, TargetConfiguration } from '@nrwl/devkit';
+
+export function createViteTargets(options: NormalizedSchema): {
+  [key: string]: TargetConfiguration;
+} {
+  return {
+    build: createBuildTarget(options),
+    serve: createServeTarget(options),
+    lint: createLintTarget(options),
+    test: createTestTarget(options),
```

Comment:
Remove this one. The Jest project generator will add it already and that's why the application.spec.ts fails. 'Test target already defined in architect schema' (Or something like that)

Reply:
Ah, cool :) Thanks!

Reply:
Just the test part. It’ll be added by the jest generator